config/settings/base.py

```diff
@@ -629,3 +629,12 @@
 MATTERMOST_ACCESS_TOKEN = env.str("MATTERMOST_ACCESS_TOKEN")
 # Team Name
 MATTERMOST_TEAM_NAME = env.str("MATTERMOST_TEAM_NAME")
+
+# WEBHOOKS
+# ------------------------------------------------------------------------------
+# Webhook username (Used with basic auth)
+WEBHOOK_USERNAME = env.str("WEBHOOK_USERNAME")
+# Webhook password (Used with basic auth)
+WEBHOOK_PASSWORD = env.str("WEBHOOK_PASSWORD")
```

Comment:
In case the `WEBHOOK_USERNAME` and `WEBHOOK_PASSWORD` indicates access to the crafty bot application, then it should be a `WEBHOOK_TOKEN` (or `CRAFTY_BOT_TOKEN` may be better) and the requested should have `headers={"Authorization": f"Token {CRAFTY_BOT_TOKEN}"}` 😇